package.json

```diff
@@ -31,6 +31,7 @@
     "build": "npm run lint && npm run build:js",
     "lint": "eslint src",
     "format": "prettier --write src",
+    "format:ci": "prettier --check src",
```

Comment:
can we call it `lint:format` or `format:check`? to make it clear its a check and can be run locally too